examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Look at the each site's start up kit. We see\n",
+    "\n",
+    "```\n",
+    "startup\n",
+    "│   │   ├── client.crt  <-- certificate\n",
+    "│   │   ├── client.key  <-- private key\n",
+    "│   │   ├── fed_client.json\n",
+    "│   │   ├── rootCA.pem  <-- root certificate\n",
+    "│   │   ├── signature.json\n",
+    "│   │   ├── start.sh\n",
+    "│   │   ├── stop_fl.sh\n",
+    "│   │   └── sub_start.sh\n",
+    "\n",
+    "```\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authorization\n",
+    "\n",
+    "Federated learning is conducted over computing resources owned by different organizations. Naturally these organizations have concerns about their computing resources being misused or abused. Even if an NVFLARE docker is trusted by participating orgs, researchers can still bring their own custom code to be part of a study (BYOC), which could be a big concern to many organizations. In addition, organizations may also have IP (intellectual property) requirements on the studies performed by their own researchers.\n",
+    "\n",
+    "Some framework disables the BYOC feature and only allowed the pre-installed package during production, but in research and developement period, researchers needs to modify the code constantly during experiments. \n",
+    "\n",
+    "\n",
+    "NVFLARE comes with an authorization system that can help address these security concerns and IP requirements. With this system, an organization can define strict policy to control access to their computing resources and/or FL jobs.\n",
+    "\n",
+    "Here are some examples that an org can do:\n",
+    "\n",
+    "* Restrict BYOC to only the org’s own researchers;\n",
+    "\n",
+    "* Allow jobs only from its own researchers, or from specified other orgs, or even from specified trusted other researchers;\n",
+    "\n",
+    "* Totally disable remote shell commands on its sites\n",
+    "\n",
+    "* Allow the “ls” shell command but disable all other remote shell commands\n",
+    "\n",
+    "\n",
+    "### Centralized vs. Federated Authorization\n",
+    "\n",
+    "Instead of relying a centralized (FL Server) to authorizate and authenticate the user, NVFLARE now uses federated authorization where each organization defines and enforces its own authorization policy:\n",
+    "\n",
+    "Each organization defines its policy in its own authorization.json (in the local folder of the workspace). This locally defined policy is loaded by FL Clients owned by the organization, The policy is also enforced by these FL Clients.\n",
+    "\n",
+    "This decentralized authorization has an added benefit: since each organization takes care of its own authorization, there will be no need to update the policy of any other participants (FL Server or Clients) when a new orgs or clients are added.\n",
```

Comment:
suggest: 
"This federated authorization has an added benefit: since each organization manages its own authorization, there is no need to update the policy of any other participants (FL Server or Clients) when new organizations or clients are added."